test/Extensions/TesterAzureUtils/Persistence/PersistenceProviderTests.cs

```diff
@@ -320,9 +321,12 @@ private async Task<AzureTableGrainStorage> InitAzureTableGrainStorage(bool useJs
             // TODO change test to include more serializer?
             var binarySerializer = new OrleansGrainStorageSerializer(this.providerRuntime.ServiceProvider.GetRequiredService<Serializer>());
             var jsonSerializer = new JsonGrainStorageSerializer(new OrleansJsonSerializer(jsonOptions));
-            options.GrainStorageSerializer = useJson
-                ? new GrainStorageSerializer(jsonSerializer, binarySerializer)
-                : new GrainStorageSerializer(binarySerializer, jsonSerializer);
+            if (useFallback)
```

Comment:
The issue I reported is not found with this unit test.
because we've used serializer has fallback (binary -> json and json -> binary).